theme/static/js/hs_file_browser.js

```diff
@@ -107,16 +109,31 @@ function updateSelectionMenuContext() {
     var flagDisableGetLink = false;
     var flagDisableCreateFolder = false;
 
+    var maxSize = MAX_FILE_SIZE * 1024 * 1024; // convert MB to Bytes
+
     if (selected.length > 1) {
         flagDisableRename = true; 
         flagDisableOpen = true;
         flagDisablePaste = true;
         flagDisableZip = true;
         flagDisableSetGeoRasterFileType = true;
         flagDisableGetLink = true;
+        
+        for (var i = 0; i < selected.length; i++) {
+            var size = parseInt($(selected[i]).find(".fb-file-size").attr("data-file-size"));
+            if (size > maxSize) {
+                flagDisableDownload = true;
+            }
+        }
+        $("#fb-download-help").toggleClass("hidden", !flagDisableDownload);
```

Comment:
If there are multiple files selected for downloading, does the logic above mean if there is one file that is larger than 1GB but other files are smaller than 1GB, then no file will be allowed to be downloaded?